server/hotrod/src/main/scala/org/infinispan/server/hotrod/ClientListenerRegistry.scala

```diff
@@ -211,16 +214,16 @@ class ClientListenerRegistry(configuration: HotRodServerConfiguration) extends L
            extends BaseClientEventSender(ch, listenerId, version, targetEventType)
 
    private abstract class BaseClientEventSender(ch: Channel, listenerId: Bytes, version: Byte, targetEventType: ClientEventType) {
-      val eventQueue = new LinkedBlockingQueue[AnyRef](100)
+      val eventQueue = new LinkedBlockingQueue[AnyRef](50)
```

Comment:
Whoops I didn't mean to change that, let me change it back.  We can cover this in a different PR if we want to make it configurable.